dist/FileAPI.html5.js

```diff
@@ -107,6 +107,7 @@
 		userAgent = window.navigator.userAgent,
 		safari = /safari\//i.test(userAgent) && !/chrome\//i.test(userAgent),
 		iemobile = /iemobile\//i.test(userAgent),
+		insecureChrome = /chrome\//i.test(userAgent) && window.location.protocol === 'http:',
```

Comment:
If into Safari is exists the same problem, then OK, otherwise need check `!safari`.